app/views/assignments/student_assignment_list.html.erb

```diff
@@ -1,8 +1,9 @@
 <% content_for :title, t('menu.home') %>
 
-<% upcoming, past = Assessment.where(is_hidden: false)
+<% upcoming, past = @current_user.visible_assessments()
```

Comment:
You don't need the `()` after visible assessments here. Then align the next few lines so the `.` lines up